### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by nr-ahemsath**
Adds standard issue templates and updates the bug report template with .NET agent specific details.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
-
-* ex: Browser name and version:
-* ex: Operating System and version:
+**Steps to Reproduce**
+[NOTE]: # ( Please be as specific as possible. )
+[TIP]:  # ( Link a sample application that demonstrates the issue. )
 
-## Additional context
+**Your Environment**
+[TIP]:  # ( Include as many relevant details about your environment as possible including the running version of the agent and any relevant configurations, such as which .NET framework or core version your apllication is running on. )
```
Comment (lspangler584) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: "such as which .NET framework or core version your apllication is running on. )"
=>
"such as the .NET framework or core version of your application.)"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+priority: '' 
+---
+
+### Is your feature request related to a problem? Please describe.
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (lspangler584) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: "A clear and concise description of what the problem is."
=>
"A clear and concise description of the problem."

**Comment by lspangler584** [comment]
Do we not need `enhancement.md` or is that covered elsewhere?

**Comment by nr-ahemsath** [comment]
> Do we not need `enhancement.md` or is that covered elsewhere?

It got renamed to `feature_request.md`.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (nr-ahemsath) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Good suggestion.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (nr-ahemsath) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good idea, and thanks for catching my typo.  "Apllication?" 🤦

**Pull request merged**